src/Sulu/Bundle/ContentBundle/Repository/NodeRepository.php

```diff
@@ -635,15 +635,28 @@ public function getNodesTree(
      * @param mixed $locale
      * @param mixed $excludeGhost
      * @param mixed $complete
+     *
+     * @return array
      */
     private function loadNodeAndAncestors($uuid, $webspaceKey, $locale, $excludeGhost, $complete)
     {
-        $descendants = $this->getMapper()->loadNodeAndAncestors($uuid, $locale, $webspaceKey, $excludeGhost);
+        $descendants = $this->getMapper()->loadNodeAndAncestors(
+            $uuid,
+            $locale,
+            $webspaceKey,
+            $excludeGhost,
+            $excludeGhost
```

Comment:
yes ... but i could also introduce a new parameter for the controller (`$excludeShadow`) and pass this to this functions ...